doc/quickstart.rst

```diff
@@ -0,0 +1,35 @@
+##########
+Quickstart
+##########
+
+.. raw:: html
+
+   <div class="interactive-tour-container">
+      <img id="tour-default-layout-main" class="tour-screenshot tour-screenshot-active" src="_static/tour/default-layout-main.png">
+      <img id="tour-options-menu" class="tour-screenshot" src="_static/tour/options-menu.png">
+      <img id="tour-context-menu" class="tour-screenshot" src="_static/tour/context-menu.png">
+      <img id="tour-variable-explorer" class="tour-screenshot" src="_static/tour/variable-explorer.png">
+      <img id="tour-plots" class="tour-screenshot" src="_static/tour/plots.png">
+      <img id="tour-files" class="tour-screenshot" src="_static/tour/files.png">
+      <img id="tour-find" class="tour-screenshot" src="_static/tour/find.png">
+      <img id="tour-profiler" class="tour-screenshot" src="_static/tour/profiler.png">
+      <img id="tour-code-analysis" class="tour-screenshot" src="_static/tour/code-analysis.png">
```

Comment:
This code is correct, but the image files names are swapped (pr`profiler.png` is the code analysis and vice versa), and so in the tour itself they are reversed.

Reply:
Fixed, thanks!